rfcs/rfc-template.md

```diff
@@ -0,0 +1,59 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`(dcp-community/rfc#)[https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>]`
+
+# RFC Name
+
+*This is the same as the filename.*
+
+## Summary
+
+*Describe the vision of the RFC in 2-3 sentences. Consider this section as your "elevator pitch" or "release note" to the community.*
+
+## Author(s)
+
+*Recommended format for Authors:*
+
+ `[Name](mailto:username@example.com)`
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+*Describe the user or technical need in detail [with alignment to the DCP **Thematic Roadmap** where possible]. Reference prior community discussions to demonstrate support for this RFC.*
+
+### User Stories or Use Cases [optional]
```

Comment:
I would make this not optional since I think it's important to explain who this is for (helps target folks for comments on the RFC as well)